app/assets/javascripts/application.js.erb

```diff
@@ -515,16 +465,41 @@ const onPageLoad = (callback) => {
 const replace = (oldElement, newHTMLString) => {
   const placeHolder = document.createElement('div');
   placeHolder.insertAdjacentHTML('afterBegin', newHTMLString);
-  const newElement = placeHolder.firstElementChild;
-  oldElement.replaceWith(newElement);
+  const newElements = placeHolder.childNodes.values();
```

Comment:
Ah, yeah, when `app/views/comments/_comment.html.erb` renders it's two HTML elements, the `<input>` for the comment folder and then the `<div>` for the comment.